src/localforage.js

```diff
@@ -118,50 +118,58 @@ class LocalForage {
         // If the options argument is an object, we use it to set values.
         // Otherwise, we return either a specified config value or all
         // config values.
-        if (typeof options === 'object') {
-            // If localforage is ready and fully initialized, we can't set
-            // any new configuration values. Instead, we return an error.
-            if (this._ready) {
-                return new Error(
-                    "Can't call config() after localforage " + 'has been used.'
-                );
-            }
-
-            for (let i in options) {
-                if (i === 'storeName') {
-                    options[i] = options[i].replace(/\W/g, '_');
+        try {
+            if (typeof options === 'object') {
+                // If localforage is ready and fully initialized, we can't set
+                // any new configuration values. Instead, we return an error.
+                if (this._ready) {
+                    return new Error(
+                        "Can't call config() after localforage " + 'has been used.'
+                    );
                 }
 
-                if (i === 'version' && typeof options[i] !== 'number') {
-                    return new Error('Database version must be a number.');
+                for (let i in options) {
+                    if (i === 'storeName' && typeof options[i] !== 'string') {
+                        options[i] = options[i].replace(/\W/g, '_');
+                    } else {
+                        return new Error('storeName must be a string');
```

Comment:
This is a helpful touch, and I guess it isn't a breaking change since `.replace` on anything that isn't a `String` would fail anyway (since it isn't a function).